### ISSUE TITLE
dropout

### ISSUE THREAD

**Issue by MortezaMardani** [lucidrains/denoising-diffusion-pytorch#73]
the training for the current code on cifar10 is overfitting. It doesn't get any good FID close to the numbers reported in the paper. It seems the paper uses dropout with rate=0.1 that is missing in the code. where should be added in Unet?
